test/controllers/trips_controller_test.rb

```diff
@@ -1,23 +1,80 @@
 require "test_helper"
 
+# TODO: update trip tests
+
 describe TripsController do
-  describe "show" do
-    # Your tests go here
-  end
 
-  describe "create" do
-    # Your tests go here
+  let (:driver) {
+    Driver.create name: "Captain Quint",
+      vin: "398YEB947GH3452BH",
+      available: true
+  }
+
+  let (:passenger) {
+    Passenger.create name: "Matt Hooper",
+      phone_num: "345.635.2129"
+  }
+
+  let (:trip) {
+    Trip.create passenger_id: passenger.id,
+      driver_id: driver.id,
+      date: Date.today,
+      rating: nil,
+      cost: 12
+  }
+  
+  describe "index" do
+    it "returns a list of all trips in the database" do
+      get "/trips"
+      must_respond_with :success
+    end 
   end
 
-  describe "edit" do
-    # Your tests go here
+  describe "show" do 
+    it "responds with success for an existing valid trip id" do
+      get trip_path(passenger.id)
+      must_respond_with :success
+    end
+
+    it "redirects to root_path if a trip is nil" do
+      get "/trips/12500"
+      must_redirect_to root_path
+    end
   end
 
   describe "update" do
-    # Your tests go here
+    it "returns an error if trip is nil" do
+      get "/trip/1876"
```

Comment:
Similar to other test: consider using the named path helpers instead (`get trip_path(1876)`)

This test is failing because it's weird: There IS no path for `/trip/:id` (note singular...) it's `/trips/:id`. One good reason to use the named path helpers. :)